core/templates/pages/admin-page/roles-tab/role-graph.directive.ts

```diff
@@ -33,7 +33,7 @@ angular.module('oppia').directive('roleGraph', [
         //            'source': id of source node
         //            'target': id of target node
         //  - 'initStateId': The initial state id
-        //  - 'finalStateIds': The list of ids corresponding to terminal states
+        //  - 'finalStateIds': The list of ids corresponding to terminal states.
```

Comment:
Can you please add a period after each line above?

Reply:
Done.